galley/pkg/kube/source/source.go

```diff
@@ -31,6 +34,38 @@ import (
 
 var scope = log.RegisterScope("kube", "kube-specific debugging", 0)
 
+const convertEnvVar = "ISTIO_CONVERT_K8S_SERVICE"
+
+func enabledKubeTypes() ([]kube.ResourceSpec, error) {
+	x := os.Getenv(convertEnvVar)
+	if x == "" {
+		x = "false"
+	}
+	b, err := strconv.ParseBool(x)
+	if err != nil {
+		return nil, fmt.Errorf("the value of %s is invalid: %v", convertEnvVar, err)
+	}
+
+	allTypes := kube_meta.Types.All()
+	if b {
+		return allTypes, nil
+	}
+	var filteredTypes []kube.ResourceSpec
+	for _, t := range allTypes {
+		if t.Kind != "Service" {
```

Comment:
no hardcoding "Service"

Reply:
I didn't find any package that would define those constants (and k8s code itself seems to use string literals all over the place)